config/ksql-server.properties

```diff
@@ -58,6 +58,9 @@ ksql.logging.processing.stream.auto.create=true
 # The set of Kafka brokers to bootstrap Kafka cluster information from:
 bootstrap.servers=localhost:9092
 
+# uncomment the below to point to a Schema Registry cluster
+# ksql.schema.registry.url=http://localhost:8081
```

Comment:
This conflicts/duplicates https://github.com/confluentinc/ksql/blob/86bf19099774476cf4b5e569e62bc3fde9031bc7/config/ksql-server.properties#L68, and similarly for the production configs in the file above.